Gemfile.lock

```diff
@@ -0,0 +1,88 @@
+GEM
```

Comment:
@mkudlej Gemfile.lock freezes all the versions required for the app, if we ignore that then latest versions of the gems will be installed and might cause conflicts and issues which are unforeseen.